python/ray/train/v2/_internal/execution/controller/controller.py

```diff
@@ -110,7 +112,8 @@ def __init__(
         callbacks: Optional[List[RayTrainCallback]] = None,
     ):
         self._train_run_context = train_run_context
-        configure_controller_logger(self._train_run_context)
+        if ray_constants.env_bool(ENABLE_CONTROLLER_LOGGING_ENV_VAR, True):
+            configure_controller_logger(self._train_run_context)
```

Comment:
nit: make the default value a constant `DEFAULT_ENABLE_CONTROLLER_LOGGING = True`

Reply:
Done.